### PR TITLE
Add types field to package.json

### PR DESCRIPTION
**Pull request by sanjom**
This allows TypeScript to find the type definitions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.12.1",
   "description": "An advanced barcode-scanner written in JavaScript",
   "main": "lib/quagga.js",
+  "types": "./type-definitions/quagga.d.ts",
   "browser": "dist/quagga.min.js",
```
Comment (sharikovvladislav) [inline package.json]: why do you put `./` at the start? browser and main fields do not have same, why you add it there?

Comment (sanjom) [reply package.json]: yes, should be without the `./` for consistency

**Comment by sharikovvladislav** [comment]
By the way, current typings do not work with latest examples. 

Check this: https://github.com/serratus/quaggaJS/blob/master/example/live_w_locator.js#L234
And this: https://github.com/serratus/quaggaJS/blob/master/type-definitions/quagga.d.ts#L309

So current typings do not support new examples.

**Comment by robingenz** [comment]
any updates?

**Comment by ericblade** [comment]
@robingenz i believe this is already in the repo

**Pull request closed without merging**